guide/getting-started/software.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: guide
+title: Software overview
+nav_order: 11
+parent: Getting started
+permalink: /guide/getting-started/software/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+This page highlights some of the most common software categories. Future expansion
+could be to create sub-pages for each category to discusss it in more detail.
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/getting-started/software/software-overview.jpg"
+   retina = "/assets/images/guide/getting-started/software/software-overview@2x.jpg"
+   mobile = "/assets/images/guide/getting-started/hardware/software-overview-mobile.jpg"
+   mobileRetina = "/assets/images/guide/getting-started/software/software-overview-mobile@2x.jpg"
+   alt-text = ""
+   caption = 'Texture by [Bilal O.](https://unsplash.com/@lightcircle){:target="_blank" rel="nofollow"} on [Unsplash](https://unsplash.com){:target="_blank" rel="nofollow"}.'
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Software overview
+
+Finance is complex and there are many use cases, from giving weekly allowance to a child and selling goods in a store to managing multi-billion dollar retirement funds and hectic day trading. So we end up with many different applications solutions for the various uses cases. Bitcoin being a global currency adds another layer of complexity as the software may need to be adapted to different languages, cultures and legal frameworks. When designing, it's a good idea to have a feel for this landscape and know where your product fits. Here is an overview of several common and specialized categories.
```

Comment:
I think it would be:
"So we end up with many different applications..."
or
"So we end up with many different application solutions..."

either way, it is "...for the various use cases" at the end.

Reply:
You can make suggested changes which are handy for typos and grammar - https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request

Reply:
Hm, that's how I thought I did it....

Reply:
Added the changes. Thank you.